.github/workflows/publish_image_chart.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
           - image: launcher-okd
             dockerfile: cmd/launcher/Dockerfile-okd
           - image: pipecd
-            dockerfile: cmd/pipecd/Dockerfile
+            dockerfile: cmd/control-plane/Dockerfile
```

Comment:
```suggestion
            dockerfile: cmd/control-plane/Dockerfile
          - image: control-plane
            dockerfile: cmd/control-plane/Dockerfile
```
nits

Reply:
Aka. Build both pipecd image and control-plane image based on `cmd/control-plane/Dockerfile`